Klarna.php

```diff
@@ -3651,13 +3651,24 @@ protected function xmlrpc_call($method, $array)
                 }
             }
 
-            //Send the message.
             $selectDateTime = microtime(true);
             if (self::$xmlrpcDebug) {
                 $this->xmlrpc->setDebug(2);
             }
+            //Save previous XML RPC client encoding settings.
+            $tmp_xmlrpc_defencoding = $GLOBALS['xmlrpc_defencoding'];
+            $tmp_xmlrpc_internalencoding = $GLOBALS['xmlrpc_internalencoding'];
+            //Set XML RPC client encoding settings.
+            $GLOBALS['xmlrpc_defencoding'] = 'ISO-8859-1';
+            $GLOBALS['xmlrpc_internalencoding'] = 'UTF-8';
```

Comment:
This feels a bit odd.. isn't this the purpose of the [$accepted_charset_encodings](https://github.com/gggeek/phpxmlrpc/blob/master/lib/xmlrpc.inc#L856) and [$request_charset_encoding](https://github.com/gggeek/phpxmlrpc/blob/master/lib/xmlrpc.inc#L858) fields?

```php
	// The charset encoding used by the server for received messages and
	// by the client for received responses when received charset cannot be determined
	// or is not supported
	$GLOBALS['xmlrpc_defencoding']='UTF-8';

	// The encoding used internally by PHP.
	// String values received as xml will be converted to this, and php strings will be converted to xml
	// as if having been coded with this
	$GLOBALS['xmlrpc_internalencoding']='ISO-8859-1';
```

Those globals seems to only be there as a fallback.